src/Compilers/Core/Portable/CodeGen/ITokenDeferral.cs

```diff
@@ -21,6 +21,8 @@ internal interface ITokenDeferral
 
         Cci.IFieldReference GetArrayCachingFieldForConstants(ImmutableArray<ConstantValue> constants, Cci.IArrayTypeReference arrayType, SyntaxNode syntaxNode, DiagnosticBag diagnostics);
 
+        Cci.IFieldReference? GetFieldForDataString(string text, ImmutableArray<byte> data, SyntaxNode syntaxNode, DiagnosticBag diagnostics);
```

Comment:
>ImmutableArray<byte> data [](http://example.com/codeflow?start=64&length=25)

It looks like the ```data``` parameter is redundant. Its value is defined by content of the ```text``` parameter. Passing redundant data is a possible source of bugs because it becomes responsibility of each consumer to keep data in sync. Not to mention the fact that each consumer is responsible to do more work.